helm-charts/splunk-otel-collector/templates/secret.yaml

```diff
@@ -26,4 +26,13 @@ data:
   {{- with .Values.splunkPlatform.caFile }}
   splunk_platform_hec_ca_file: {{ . | b64enc }}
   {{- end }}
+  {{- with .Values.agent.etcd.caCert }}
+  etcd_ca_cert: {{ . | b64enc }}
+  {{- end }}
+  {{- with .Values.agent.etcd.clientCert }}
+  etcd_client_cert: {{ . | b64enc }}
+  {{- end }}
+  {{- with .Values.agent.etcd.clientKey }}
+  etcd_client_key: {{ . | b64enc }}
+  {{- end }}
```

Comment:
I'm not sure we should mix Splunk related secrets with etcd certificates. Maybe better to create another k8s secret just for etcd?

Reply:
I agree. Done.